lib/jxl/enc_fast_lossless.cc

```diff
@@ -19,18 +19,24 @@
 
 #if !FJXL_STANDALONE
 #include "lib/jxl/encode_internal.h"
-#endif
+#endif  // !FJXL_STANDALONE
+
+#if defined(JXL_HAVE_BUILTIN_CPU_SUPPORTS)
+#define FJXL_BUILTIN_CPU_SUPPORTS(F) __builtin_cpu_supports(F)
+#else  // JXL_HAVE_BUILTIN_CPU_SUPPORTS
+#define FJXL_BUILTIN_CPU_SUPPORTS(F) 1
```

Comment:
I am not sure I like this ;)

Reply:
Unfortunately we have to avoid `__builtin_cpu_supports` if compiler-rt library does not provide `___cpu_model`.

We can borrow some code from HWY targets.cc... WDYT?

Reply:
That seems a lot better :-)

Also, I would probably flip the logic here - i.e. have a JXL_MISSING_BUILTIN_CPU_SUPPORTS macro, so that if you do nothing you get the standard builtin.

Reply:
SGTM. Will do soon.

Reply:
Done. Waiting for CI.